backend/app/controllers/spree/admin/base_controller.rb

```diff
@@ -62,6 +62,28 @@ def lock_order
       def order_mutex_redirect_path
         edit_admin_order_path(@order)
       end
+
+      def add_breadcrumb(name, path = nil, render_options = {})
```

Comment:
I'm glad that you make this suggestion because I agree a concern here is a nice way to keep code organised. The reason I didn't use one is that I found that the `backend` gem doesn't use concerns at all for any other purpose so I wasn't sure using one here would be well received.

I'll move these into a concern right away!